webpack.dev.config.js

```diff
@@ -22,7 +22,9 @@ module.exports = {
     filename: './index.js',
     path: path.resolve(__dirname),
     library: '@iterable/web-sdk',
-    libraryTarget: 'umd'
+    libraryTarget: 'umd',
+    hotUpdateChunkFilename: 'hot/hot-update.js',
+    hotUpdateMainFilename: 'hot/hot-update.json'
```

Comment:
These had been removed in https://github.com/Iterable/iterable-web-sdk/commit/eb9d23e56083fbe897efa2e5a1d0e545f99b388e#diff-03651daa986b59c8421cba91a0ff6e7ffdfc40623e571f6525f648c2a6e19308 but need to put them back so that hot-update files don't get generated in the root level and tracked by git